assets/javascripts/wizard/components/field-validators.js.es6

```diff
@@ -0,0 +1,9 @@
+import Component from "@ember/component";
+import { observes } from "discourse-common/utils/decorators";
+export default Component.extend({
+    actions:{
+        perform() {
+            this.appEvents.trigger('custom-wizard:validate');
```

Comment:
@fzngagan This event structure is cool as it allows you to use the perform action in the validators regardless of where they are, however I'm not sure it's used in the suggested-validator? Could you explain the thinking here a bit? Let's discuss.